curation/views.py

```diff
@@ -138,12 +141,62 @@ def _str_to_int(value):
         gene=gene,
     )
     if genomic_coords:
-        response['gene'] = genomic_coords.get("gene", None)
-        response['chr'] = genomic_coords.get("chr", None)
+        gene = genomic_coords.get('gene', None)
+        response['gene'] = gene
```

Comment:
Out of curiosity, why did `response['gene'] = genomic_coords.get("gene", None)` not work?

Reply:
Not too important, `gene` just gets used on both of the next lines so setting it to a separate variable saves a small function call

Reply:
Ahhh missed that! Cool